### ISSUE TITLE
Attachment drag drop issue

### ISSUE THREAD

**Issue by angadsalaria** [wolfgangimig/ddaddin#7]
We are experiencing issues around drag-drop of attachments in outlook emails. The issue occurs intermittently.

When dragging files from outlook over to file system (or HTML web page for that matter), quite frequently, the files are copied over as empty files. 

I inspected the user's "temp" folder and found the below for such cases:
- User invokes Drag of attachment file from outlook email:
  
  An empty file with the same name and type is created in "temp" folder.
- User drops the file to a folder: 
  
  The empty file from "temp" folder is copied over to the drop location. And the drop action initiates a complete file transfer of the original file to the "temp" folder. However, the dropped file is still an empty file.

The same file copies over successfully the next time it is dragged (since a complete copy is already available in temp folder).

I used the demo version to carry out this test case. I've attached the logs below. Look for logs around the file "Resources.docx". I drag-dropped this file multiple times: 
[ddaddin.zip](https://github.com/wolfgangimig/ddaddin/files/68800/ddaddin.zip)

**Comment by wolfgangimig** [comment]
This effect could be caused by a virus scanner. Please check whether this problems also occurs when you disable the virus scanner.

**Comment by angadsalaria** [comment]
The problem does not occur when disabling the virus scanner.

**Comment by wolfgangimig** [comment]
Thanks for your reply. Which anti virus product do you use? I hope you found an appropriate configuration option to prevent the AV from scanning the attachments.

**Comment by angadsalaria** [comment]
Can't disclose the anti-virus product owing to company policy, but we did figure out that the issue was limited to Office 2007. I know you did not make a mention that the add-in would work in 2007, but we tried it nevertheless and found that drag-drop for email messages worked fine. Attachments gave intermittent issues with the anti-virus, and turning the anti-virus off did resolve the issue.

We upgraded office and tested the addin with the same anti-virus and had no issues.

Thanks Wolfgang.
